client/app/homepage/homepage.module.ts

```diff
@@ -0,0 +1,19 @@
+import { NgModule }      from '@angular/core';
+import { CommonModule }  from '@angular/common';
+
+import { RouterModule, Router } from '@angular/router';
+import { HomePage } from './homepage.component';
+
+import { FormsModule } from '@angular/forms';
+
+export const routes = [
+  { path: '', component: HomePage, pathMatch: 'full' }
+];
+
+@NgModule({
+  imports: [ CommonModule, RouterModule.forChild(routes), FormsModule ],
+  declarations: [ HomePage ]
+})
+export class HomePageModule {
+  static routes = routes;
```

Comment:
This can be removed. It is required in the case where the routes have to be configured int he module class and is put static so thet all the instances of this modulle use the same soute properties